mesheryctl/internal/cli/root/app/app.go

```diff
@@ -67,19 +68,26 @@ func init() {
 }
 
 func getSourceTypes() error {
+	log, err := logger.New("app", logger.Options{
+		Format:     logger.SyslogLogFormat,
+		DebugLevel: true,
+	})
 	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 	if err != nil {
-		return err
+		log.Error(err)
```

Comment:
@suhail34 What's the reason for using `log.Error(err)` instead of `return err` ?

Reply:
It returns the probable cause ,remediation, long descrip, short descrip, and code which normally returning doesn't do

Reply:
@suhail34 can you paste an example  screenshot

Reply:
Is this resolved?